code/modules/mapping/map.dm

```diff
@@ -158,6 +158,10 @@
 	/// force world to be bigger height
 	var/world_height
 
+	/// allow random picking if no map set
+	/// used to exclude indev maps
```

Comment:
use means set to false???

Reply:
basically, 'set to false to prevent travis'

this is so people don't forget to turn stuff on; random failures are bad but so is not having any map coverage